app/server/lib/DocApi.ts

```diff
@@ -87,6 +87,7 @@ import {downloadDSV} from "app/server/lib/ExportDSV";
 import {collectTableSchemaInFrictionlessFormat} from "app/server/lib/ExportTableSchema";
 import {streamXLSX} from "app/server/lib/ExportXLSX";
 import {expressWrap} from 'app/server/lib/expressWrap';
+import { filenameContentDisposition } from "app/server/lib/filenamesUtils";
```

Comment:
Please keep import styling consistent with those around them. In this case, removing spaces between { and }.

Reply:
Ok but unfortunately `yarn lint` does not include this rule.